### ISSUE TITLE
Can't access pihole Web interface 

### ISSUE THREAD

**Issue by batrarobin** [pi-hole/pi-hole#1499]
Hello, 

I have read FAQ and some of the similar issues but not being able to resolve the issue. 

I have installed pihole through DietPi on my Pi Zero.  I am not being able to access the web interface of PiHole.  My Router is R8000 with DDWRT installed. 

* I have run the commands w.r.t lighttpd force-reload etc but with no luck. 

* nslookup doubleclick.net and nslookup pi.hole correctly show the IP address of the Pi (192.168.1.111) 

* Ipv6 is disabled on router. 

My log debug token is o2rg3knmna

Could you pls help. Thanks.

**Comment by PromoFaux** [comment]
If you use http://pi.hole/admin does that work?

**Comment by Mcat12** [comment]
If you installed via DietPi, then the web interface may be located at http://pi.hole/pihole

**Comment by batrarobin** [comment]
@Mcat12 @PromoFaux

Tried via both links but no luck. The error is: pi.hole’s server DNS address could not be found.

**Comment by Mcat12** [comment]
Try using the IP address instead. That error means that the device isn't able to contact the Pi-hole (isn't using it for DNS).

**Comment by technicalpyro** [comment]
@Mcat12  this is also the case when dnsmasq is still starting so if blocking ins't working either may be a bigger issue?

**Comment by batrarobin** [comment]
@technicalpyro 
@Mcat12
@PromoFaux

Thank you for your replies, I tried different settings but still with no luck. The blocking is showing as enabled but the ads are still showing on Web pages. Here are my settings so far:





![screenshot_2017-05-27-13-48-51-912_com android chrome](https://cloud.githubusercontent.com/assets/24936197/26518463/e1341a0c-42e3-11e7-95f7-911d9a4445aa.png)
![screenshot_2017-05-27-13-39-04-789_com android chrome](https://cloud.githubusercontent.com/assets/24936197/26518464/e134f7ba-42e3-11e7-88da-802fbf521889.png)
![screenshot_2017-05-27-13-48-30-394_com android chrome](https://cloud.githubusercontent.com/assets/24936197/26518465/e1351e48-42e3-11e7-9b2a-b63c9e1b597e.png)
![screenshot_2017-05-27-13-39-18-659_com android chrome](https://cloud.githubusercontent.com/assets/24936197/26518466/e13a8b76-42e3-11e7-9b5a-e1e290bc00f7.png)
![img_20170527_134717_hht](https://cloud.githubusercontent.com/assets/24936197/26518480/f33a462c-42e3-11e7-95f7-d67edc83a516.jpg)
![img_20170527_134651_hht](https://cloud.githubusercontent.com/assets/24936197/26518481/f33b4c2a-42e3-11e7-8c62-57e414a14c05.jpg)

**Comment by batrarobin** [comment]
![img_20170527_135722_hht](https://cloud.githubusercontent.com/assets/24936197/26518502/82e16a30-42e4-11e7-8407-f376287eff43.jpg)

**Comment by Mcat12** [comment]
Can your device ping the Pi-hole's IP address and vice versa? If not, how are the Pi-hole and devices connected to the network?

**Comment by Mcat12** [comment]
@batrarobin

**Comment by batrarobin** [comment]
@Mcat12

Apologies for not getting back any earlier, I tried various options, I wasnt able to ping the pihole device due to some reason. It looked like a bad installation of dietpi or maybe some setting at my router. I didn't have much time to troubleshoot so I temporarily put it aside, I will re-install the diet pi and give it another go. 

In the meantime I am closing this issue, if it still persists I will re-open it. 

Thanks for your help.

**Comment by chiu0602** [comment]
I have just installed pihole on dietpi, below terminal text show why the links not work.

```
root@DietPi:/var/www# ls -l
total 72
lrwxrwxrwx 1 www-data www-data    19 Aug 15 23:31 admin -> /var/www/html/admin
-rwxrwxr-x 1 www-data www-data 37913 Aug 15 23:31 apc.php
-rwxrwxr-x 1 www-data www-data    68 Aug 15 23:31 index.html
-rwxrwxr-x 1 www-data www-data 22915 Aug 15 23:31 opcache.php
-rwxrwxr-x 1 www-data www-data    20 Aug 15 23:31 phpinfo.php
lrwxrwxrwx 1 www-data www-data    19 Aug 15 23:31 pihole -> /var/www/html/admin
root@DietPi:/var/www# cat index.html
<html>
Blocked by Pi-hole.
<script>window.close();</script>
</html>
root@DietPi:/var/www#
```

**Comment by Mcat12** [comment]
Make a new issue instead of commenting on closed issues
